pkg/boot/fit/vfit.go

```diff
@@ -103,7 +103,7 @@ func (s RSASignature) Verify(b []byte, ring openpgp.KeyRing) (*bytes.Reader, err
 		return r, err
 	}
 
-	hashed, err := vfile.CalculateHash(bytes.NewReader(b), s.hash.New())
+	hashed, _ := vfile.CalculateHash(bytes.NewReader(b), s.hash.New())
```

Comment:
we either need to stop ignoring an error or at least document why we feel it is safe to ignore the error.

Reply:
see #2529